source/_integrations/librehardwaremonitor.markdown

```diff
@@ -0,0 +1,78 @@
+---
+title: Libre Hardware Monitor
+description: Instructions on how to integrate Libre Hardware Monitor within Home Assistant.
+ha_category:
+  - System monitor
+ha_release: 2025.4
+ha_iot_class: Local Polling
+ha_domain: librehardwaremonitor
+ha_platforms:
+  - sensor
+ha_integration_type: integration
+ha_quality_scale: bronze
+---
+
+The `librehardwaremonitor` {% term integration %} uses your [Libre Hardware Monitor](https://github.com/LibreHardwareMonitor/LibreHardwareMonitor) installation as a source for sensors that will display system information.
+
+## Prerequisites
+
+Libre Hardware Monitor must be running on the host, with "Remote web server" active. You also need to open inbound port (TCP 8085) on the host.
+
+To open port (on Windows):
+
+1. Navigate to Control Panel, System and Security and Windows Firewall.
+2. Select **Advanced settings** and highlight **Inbound Rules** in the left pane.
+3. Right click Inbound Rules and select New Rule.
+4. Add the port you need to open and click Next.
+5. Add the protocol (TCP) and the port number (8085) into the next window and click Next.
+6. Select Allow the connection in the next window and hit Next.
+7. Select the network type as you see fit and click Next.
+8. Name the rule and click Finish.
+
+{% include integrations/config_flow.md %}
+
+{% configuration_basic %}
+host:
+  description: IP address or hostname of the system where Libre Hardware Monitor is running.
+port:
+  description:  The port of your Libre Hardware Monitor API. Defaults to 8085.
+scan_interval:
+  description: Time between updates of data fetched from Libre Hardware Monitor. Allowed values are between 5 and 30 seconds. Defaults to 30 seconds.
+{% endconfiguration_basic %}
+
+## Configuration options
+
+In the configuration flow you can choose which hardware devices to add to Home Assistant, e.g. if you only want to monitor your CPU you can deselect any other device.
+All sensors will be grouped by the device they belong to. If you do not want all sensors for a device, you can disable entities via the UI after setup.
+
+## Reconfiguration
+
+This integration allows you to reconfigure an existing entry, e.g. if you want to add or remove hardware devices.
+
+1. Go to **{% my integrations title="Settings > Devices & services" %}**.
+2. Select the Libre Hardware Monitor integration.
+3. To reconfigure the settings, click {% icon "mdi:dots-vertical" %} to the right of an entry and select **Reconfigure**.
+
+## Known limitations
+
+Servers which require authentication are currently not supported.
+
+## Troubleshooting
+
+### Problem with connection during setup
+
+Open a browser and navigate to `http://<IP address>:<Port>` of your Libre Hardware Monitor instance, make sure you can see and update the data there.
+
+### Sensors change to "unavailable" status
+
+This is expected behavior when the computer is not reachable, e.g., it is turned off. Sensors will resume their data readings once the computer is reachable again.
```

Comment:
```suggestion
This is expected behavior when the system you are monitoring is not reachable. For example, because it is turned off. Sensors will resume their data readings once the system is reachable again.
```